packages/synapse-react-client/src/components/FeaturedResearch/FeaturedResearch.tsx

```diff
@@ -0,0 +1,361 @@
+  const url = useImageUrl(fileId ?? '', entityId)
+  if (isLoading) {
+    return <Skeleton variant="rectangular" height={142} width="100%" />
+  }
+  return (
+    <Box
+      display={'flex'}
+      gap={'30px'}
+      borderBottom={'1px solid #EAECEE'}
+      padding={'24px 0'}
+      sx={{
+        flexDirection: {
+          xs: 'column',
+          sm: 'row',
+        },
+      }}
+    >
+      <Stack useFlexGap gap={'10px'}>
+        <Typography
+          fontSize={'14px'}
+          lineHeight={'normal'}
+          fontWeight={500}
+          color={'grey.600'}
+        >
+          {research.values[affiliationColIndex]}
+        </Typography>
+        <Typography variant="headline2" color="gray.1000" fontSize={'21px'}>
+          <Link
+            href={research.values[linkColIndex] ?? ''}
+            target="_blank"
+            sx={{
+              color: 'grey.1000',
+              textDecoration: 'none',
+              '&:hover': {
+                textDecoration: 'none',
+              },
+            }}
+          >
+            {research.values[titleColIndex]}
+          </Link>
+        </Typography>
+        <Stack direction={'row'} useFlexGap gap={'16px'} alignItems={'center'}>
+          <Typography
+            variant="overline"
+            fontSize={'14px'}
+            sx={{
+              backgroundColor: 'grey.300',
+              borderRadius: '3px',
+              padding: '4px 8px',
+              lineHeight: 'initial',
+            }}
+          >
+            {parseTags(tagsColIndex, research)?.[0] || ''}
+          </Typography>
+          <Typography lineHeight={'normal'} color={'grey.600'}>
+            {research.values[publicationDateColIndex] &&
+              formatDate(
+                dayjs(Number(research.values[publicationDateColIndex])),
+                'MMMM, YYYY',
+              )}
+          </Typography>
+        </Stack>
+      </Stack>
+      <CardMedia
+        component="img"
+        image={url}
+        alt="Research Card Image"
+        sx={{
+          flexShrink: 0,
+          width: '140px',
+          height: '93.8px',
+          borderRadius: '6px',
+          objectFit: 'cover',
+          marginLeft: { xs: 'initial', sm: 'auto' },
+        }}
+      />
+    </Box>
+  )
+}
+
+const FeaturedResearchTopCard = ({
+  research,
+  entityId,
+  affiliationColIndex,
+  titleColIndex,
+  descriptionColIndex,
+  tagsColIndex,
+  linkColIndex,
+  imageColIndex,
+  isLoading,
+}: FeaturedResearchCardProps) => {
+  const fileId = research.values[imageColIndex] ?? ''
+  const url = useImageUrl(fileId || '', entityId)
+  if (isLoading) {
+    return (
+      <Skeleton
+        variant="rectangular"
+        width="100%"
+        sx={{
+          height: { xs: '250px', sm: '500px', md: '600px', lg: '800px' },
+        }}
+      />
+    )
+  }
+  return (
+    <div>
+      <CardMedia
+        component="img"
+        image={url}
+        alt="Top Research Card Image"
```

Comment:
I like that you are looking at accessibility here! Unfortunately, this is not really useful alt text ([examples](https://accessibility.huit.harvard.edu/describe-content-images)). Is it possible to have the alt text in the table row to correspond with each image? If not, I suggest we hide it from screen readers
```suggestion
        aria-hidden="true"
```